test/boost/mutation_reader_test.cc

```diff
@@ -2310,7 +2310,7 @@ SEASTAR_THREAD_TEST_CASE(test_multishard_streaming_reader) {
                 return dht::to_partition_range(*next);
             }
             return std::nullopt;
-        });
+        }, {});
```

Comment:
Do we plan on adding more tests?

Reply:
It would make sense. I just have to think of a way on how to do it.

Reply:
See https://github.com/scylladb/scylla-dtest/pull/3321